### PR TITLE
Update hardware-requirements-embed.md

### PR DESCRIPTION
**Pull request by AlonMagrafta**
quorum nodes minimal hardware requirements

### CODE REVIEW

**Comment by AlonMagrafta** [comment]
not urgent. Can be merged once approved

File: content/embeds/hardware-requirements-embed.md
Comment (djptek) [reply content/embeds/hardware-requirements-embed.md:36]: must also?

**Review by rrelledge — APPROVED** [review]

**Pull request merged**
